### DISCUSSION TITLE
Bodies with _only_ `visual` geometry --  are they supposed to participate in the simulation?

### DISCUSSION THREAD

**Discussion by rao94583** [google-deepmind/mujoco#2207] [Asking for Help]
### Intro

We're using MuJoCo for some robotics research (but then again, who isn't?)

### My setup

MuJoCo on MacOS and Ubuntu.

### My question

In the `ur5e.xml` file in `mujoco_menagerie`, there are a  `body` element (`base`)  that contains only an `inertial` element and a `geom` element of class `visual`.  I _think_ this means that only the `inertial` element interacts with the physics engine, and the geometry is only used for visualization.  And this further implies that a body must contain at least an `inertial` element, or a single non-visual `geom` element.

Am I thinking about this right?

### Minimal model and/or code that explain my question

[https://github.com/google-deepmind/mujoco_menagerie/blob/main/universal_robots_ur5e/ur5e.xml](url)

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
Not exactly.

A visual (non-colliding) geom could be used for inertia inference, and in that case would affect the physics. In the case you described, where inertia is explicit, this geom will indeed not affect the physics.

To clarify further, geoms can affect the physics in 3 unrelated ways:
1. Collisions 
2. Inertia inference.
3. Ray casting targets (e.g. rangefinder sensors)

As you see, the notion of "visual geom" which comes from URDF, is not really a helpful one and should be avoided. For example a geom that URDF would call visual will affect sensor values if a ray is cast upon it. Does that count as "affecting the physics"? 🤷🏼‍♂️
